tests/prompting/test_test.py

```diff
@@ -0,0 +1,2 @@
+def test_test():
+    assert 1 == 1
```

Comment:
why it is needed?

Reply:
it's good practice to have one test that always passes logic-wise to be able to see whether when your tests are failing it's due to code or due to issues in how unit tests are set up.